app/views/questionnaires/new.html.erb

```diff
@@ -41,63 +41,46 @@
             </div>
           </div>
         </fieldset>
-      </section>
 
-      <fieldset class="form__group">
-        <legend class="l-questionnaire__legend"><%= t('questionnaire.address.heading') %></legend>
-        <p><%= t('questionnaire.address.description') %></p>
-
-        <div class="l-questionnaire__row">
-          <div class="l-questionnaire__field">
-            <%= f.form_row :address_line_1 do %>
-                <%= f.errors_for :address_line_1 %>
-                <%= f.label :address_line_1, t('questionnaire.address.label_one'), class: 'form__label-heading' %>
-                <%= f.text_field :address_line_1 %>
-            <% end %>
-
-            <%= f.form_row :address_line_2 do %>
-                <%= f.errors_for :address_line_2 %>
-                <%= f.label :address_line_2, t('questionnaire.address.label_two'), class: 'form__label-heading' %>
-                <%= f.text_field :address_line_2 %>
-            <% end %>
-
-            <%= f.form_row :address_town do %>
-                <%= f.errors_for :address_town %>
-                <%= f.label :address_town, t('questionnaire.address.label_three'), class: 'form__label-heading' %>
-                <%= f.text_field :address_town %>
-            <% end %>
-
-            <%= f.form_row :address_county do %>
-                <%= f.errors_for :address_county %>
-                <%= f.label :address_county, t('questionnaire.address.label_four'), class: 'form__label-heading' %>
-                <%= f.text_field :address_county %>
-            <% end %>
-
-            <%= f.form_row :address_postcode do %>
-                <%= f.errors_for :address_postcode %>
-                <%= f.label :address_postcode, t('questionnaire.address.label_five'), class: 'form__label-heading' %>
-                <%= f.text_field :address_postcode %>
-            <% end %>
+        <fieldset class="form__group">
```

Comment:
I think that's because I moved a </section> tag and that section got indented. My bad.